website/docs/docs/dbt-cloud/dbt-cloud-api/service-tokens.md

```diff
@@ -1,38 +1,90 @@
 
-<Changelog>
+:::info Enterprise and Team plan differences
+Enterprise plans can apply any permission sets available to service tokens. Team plans can apply three permissions sets to service tokens: Owner, Metadata, or Job Admin.
+:::
 
- - Service Account Admin tokens were launched in February, 2021
+For more on permissions sets, see "[Enterprise Permissions](docs/dbt-cloud/access-control/enterprise-permissions)."
 
-</Changelog>
+## Generating service account tokens
 
-Service Account Admin API tokens currently have full read+write access to an account, so please use them with caution.
+In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new token. Create and save your token somewhere safe.
 
-#### Generating a Service Account Admin Token
+:::caution Note
 
-In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new Account Admin token.  Create and save your token. Please note that you will not be able to view this token again after its is generated, so store it somewhere safe for later use.
+You will not be able to view this token again after generating it, so store the token somewhere safe for later use.
 
-### Metadata Only Service Account Tokens
+:::
 
-<Changelog>
+## Permissions for service account tokens
 
- - Metadata Only Service Account tokens were launched in April, 2021
+You can assign service account tokens any permission set available in dbt Cloud. When you assign a permission set to a token, you will also be able to choose whether to grant that permissions to all projects in the account or to specific projects.
 
-</Changelog>
+### Service account tokens for Team plans
 
-dbt Cloud users can create metadata only service account based API Tokens to authorize requests to the metadata API.
+The following permissions can be assigned to a service account token on a Team plan.
 
-#### Generating a Metadata Only Service Account Token
+**Account Admin service token:**<br/>
+Account Admin service tokens have full `read + write` access to an account, so please use them with caution.  A Team plan refers to this permission set as an "Owner role."
 
-In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new Metadata Only token.  Create and save your Metadata Only token. Please note that you will not be able to view this token again after its is generated, so store it somewhere safe for later use.
+**Metadata Only service token:**<br/>
+Metadata only service tokens can authorize requests to the metadata API.
+
+**Job Admin service token:**<br/>
+Job admin service tokens can authorize requests for viewing, editing, and creating environments, triggering runs, and viewing historical runs.  
+
+**Member service token** <br/>
+Member service tokens can  authorize requests for viewing and editing resources, triggering runs, and inviting members to the account.
```

Comment:
For Member and Read-Only, should we include a link to [this page](https://docs.getdbt.com/docs/dbt-cloud/access-control/self-service-permissions) and mention something like "Tokens assigned the Member permission set will have the same permissions as a Member user"?